### DISCUSSION TITLE
Original Year isn't used for some albums

### DISCUSSION THREAD

**Discussion by hayesti** [navidrome/navidrome#3030] [Q&A] [answered]
![Screenshot 2024-05-15 153106](https://github.com/navidrome/navidrome/assets/7453861/c501f5d6-8c14-449e-9767-c9028ff0c607)

For example, for the discography of **Yes** most of the albums are correctly showing the original release year and (edition release) year. In the case of **Relayer** only the year is shown, not the original year.

In MusicBee the OriginalYear tag is populated correctly for the tracks in that album.

![Screenshot 2024-05-15 153315](https://github.com/navidrome/navidrome/assets/7453861/ed22a5cb-275f-4a60-8ae2-8013ed7fc87c)

What might be happening?

**Comment by certuna** [comment]
If you change the tag from id3v2.3 to id3v2.4, does it work?

**Reply by hayesti** [reply, marked as answer]
Yes! Or rather, the opposite. The albums displaying correctly had id3v2.3 tags whereas the one anomaly had id3v2.4 tags. When I changed the latter to id3v2.3 Navidrome displayed the album as intended.

Does this imply there is some issues with id3v2.4 tag support?
